packages/@aws-cdk/aws-stepfunctions-tasks/test/integ.glue-task.ts

```diff
@@ -0,0 +1,62 @@
+import * as glue from '@aws-cdk/aws-glue';
+import * as iam from '@aws-cdk/aws-iam';
+import * as assets from '@aws-cdk/aws-s3-assets';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import * as cdk from '@aws-cdk/core';
+import * as path from 'path';
+import * as tasks from '../lib';
+
+/*
+ * Stack verification steps:
+ * * aws stepfunctions start-execution --state-machine-arn <deployed state machine arn>
+ * * aws stepfunctions describe-execution --execution-arn <execution arn created above>
+ * * should eventually return status "SUCCEEDED"
```

Comment:
I gave this a whirl and it took nearly 15 minutes to complete. Is this normal for a simple 'hello world' glue job? (I'm not familiar with the details of aws glue) Any simple way to get this to run faster?

Reply:
Unfortunately, the cold start time for AWS Glue is about 10-15 minutes. The job itself should only take ~20 seconds, but waiting for resources can be very slow. To the best of my knowledge, there's no way to decrease the length of this delay without running a "warming job" every 10 minutes, which obviously cannot be considered for this use case. I can add a note to this effect in these steps if you think that will help.

Reply:
Ok, I see. A note there would be very useful, indeed.

Besides that, everything else LGTM. I'll approve once that note is in.